src/pyconverter/xml2py/ast_tree.py

```diff
@@ -305,7 +245,7 @@ def find(self, _type, terms=None):
             if type(item).__name__ == _type:
                 if _type == "Refname" or _type == "Refnamediv":
                     if terms == None:
-                        print("ERROR: terms not defined for a Refname class")
+                        logging.error("ERROR: terms not defined for a Refname class")
```

Comment:
```suggestion
                        logging.error("ERROR: terms are not defined for a 'Refname' class.")
```
You should have a verb in the error string and a single quote (since we can't use backticks) around the class name.